crates/nargo/src/cli/test_cmd.rs

```diff
@@ -88,13 +89,20 @@ fn run_test(
 
     // Run the backend to ensure the PWG evaluates functions like std::hash::pedersen,
     // otherwise constraints involving these expressions will not error.
-    if let Err(error) = backend.solve(&mut solved_witness, program.circuit.opcodes) {
-        let writer = StandardStream::stderr(ColorChoice::Always);
-        let mut writer = writer.lock();
-        writer.set_color(ColorSpec::new().set_fg(Some(Color::Red))).ok();
-        writeln!(writer, "failed").ok();
-        writer.reset().ok();
-        return Err(error.into());
+    match backend.solve(&mut solved_witness, &mut blocks, program.circuit.opcodes) {
+        Ok((unresolved_opcodes, oracles)) => {
+            if !unresolved_opcodes.is_empty() || !oracles.is_empty() {
+                todo!("Add oracle support to nargo test")
```

Comment:
Lets open an issue for this

Reply:
https://github.com/noir-lang/noir/issues/1052